Adding Lion:Bit S3 STEM Dev Board  to boards.txt

PradeepKotu opened the pull request:
lionbit.cc has developed new board call Lion:Bit S3 STEM Dev Board using esp32s3 add this to board file

-----------
## Description of Change
Lion:Bit S3 STEM Dev Board adding

## Tests scenarios
Tested and works fine
 
## Related links
lionbit.cc

VojtechBartoska commented:
hello @PradeepKotu, please change your PR title to more specific one. Also double check your commit, there is a lot of changes so probably something went wrong. Thanks

PradeepKotu commented:
@VojtechBartoska I have done some changes. can you check is there  any errors.

PradeepKotu commented:
because of the issue i am closing this

PradeepKotu commented:
@P-R-O-C-H-Y I have resubmitted new one and closed this. please help me for this our LionBit S3 board is already in  the market.

PradeepKotu commented:
Dear Jan Procházka

I have added a new request Number is #8552. Can you check this and please
guide me. Our board is already in  the market.   [image: WhatsApp Image
2023-08-16 at 18.11.31.jpg]


On Tue, Aug 22, 2023 at 5:34 PM Jan Procházka ***@***.***>
wrote:

> @PradeepKotu <https://github.com/PradeepKotu> Hi, It seems like you
> removed and added the boards.txt file back and git thinks that there are
> changes on every line so its hard to review the changes. Can you try to fix
> it? If needed you can close this Rand open a new clear one and just ping me
> there :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/espressif/arduino-esp32/pull/8475#issuecomment-1688055163>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIUD2IE2QP7FNHZVIZRUXS3XWSN5RANCNFSM6AAAAAA267HZXA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*Best Regards,*

<https://mvp.support.microsoft.com/profile=F588DC6D-BCA6-43D9-A903-6F5FC88F5C29>*Awarded
Winning Author*
            *Pradeep Kavinda Kotuwegedara **- (Justice of Peace - Whole
Island)*
            (BSc (Hons) in IT, MCT, MCSD, MCSA, MCSE, ACA,Office Master,
MVP Office System/Office 365)

www.Kotuwegedara.com
www.TuitionPanthiya.lk
 077-2348985 , 077-4747119

P-R-O-C-H-Y commented:
@PradeepKotu your new PR was the same, so it was closed. Idk what are you doing wrong.
Please create a new clean fork of this repository, copy the changes (just the new lines of new board) to boards.txt file and create under variants folder create a new folder with your board name and add a proper pins_arduino.h file inside. 
You can take a look how other boards are defined and the best is to use our official boards as a template :)

PradeepKotu commented:
HI
i am adding details to boards.txt file lastline or is there any way to add
only myboard details? When I add details to the end of the file it shows
more updates. What do I need to do for this issue?

On Wed, Aug 23, 2023 at 3:47 PM Jan Procházka ***@***.***>
wrote:

> @PradeepKotu <https://github.com/PradeepKotu> your new PR was the same,
> so it was closed. Idk what are you doing wrong.
> Please create a new clean fork of this repository, copy the changes (just
> the new lines of new board) to boards.txt file and create under variants
> folder create a new folder with your board name and add a proper
> pins_arduino.h file inside.
> You can take a look how other boards are defined and the best is to use
> our official boards as a template :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/espressif/arduino-esp32/pull/8475#issuecomment-1689694120>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIUD2IESUXGDE3SJHP7RTKLXWXKFPANCNFSM6AAAAAA267HZXA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*Best Regards,*

<https://mvp.support.microsoft.com/profile=F588DC6D-BCA6-43D9-A903-6F5FC88F5C29>*Awarded
Winning Author*
            *Pradeep Kavinda Kotuwegedara **- (Justice of Peace - Whole
Island)*
            (BSc (Hons) in IT, MCT, MCSD, MCSA, MCSE, ACA,Office Master,
MVP Office System/Office 365)

www.Kotuwegedara.com
www.TuitionPanthiya.lk
 077-2348985 , 077-4747119
